indexer/src/events.rs

```diff
@@ -6,39 +6,50 @@ use iota_types::{base_types::IotaAddress, event::Event};
 use serde::{Deserialize, Serialize};
 
 pub(crate) enum IotaNamesEvent {
+    // `iota-names`
     IotaNamesRegistry(IotaNamesRegistryEvent),
     IotaNamesReverseRegistry(IotaNamesReverseRegistryEvent),
+    // `auctions`
     AuctionStarted(AuctionStartedEvent),
     AuctionBid(AuctionBidEvent),
     AuctionExtended(AuctionExtendedEvent),
     AuctionFinalized(AuctionFinalizedEvent),
+    // `subdomains`
+    NodeSubdomainCreated(NodeSubdomainCreatedEvent),
+    NodeSubdomainBurned(NodeSubdomainBurnedEvent),
+    LeafSubdomainCreated(LeafSubdomainCreatedEvent),
+    LeafSubdomainRemoved(LeafSubdomainRemovedEvent),
 }
 
 impl IotaNamesEvent {
     pub(crate) fn try_from_event(
         event: &Event,
-        config: &IotaNamesConfig,
+        _config: &IotaNamesConfig,
     ) -> anyhow::Result<Option<Self>> {
         // TODO allow more package IDs
```

Comment:
If you don't have any check, the comment should be changed
```suggestion
        // TODO check for package IDs
```